WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -80,8 +80,10 @@ public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc
 
 		var rpcStatus = await rpc.GetRpcStatusAsync(cancel).ConfigureAwait(false);
 
-		var (sanityFeeRate, minEstimations) = GetFeeEstimationsFromMempoolInfo(mempoolInfo);
-
+		var minEstimations = GetFeeEstimationsFromMempoolInfo(mempoolInfo);
+		var minEstimationFor260Mb = new FeeRate((decimal)minEstimations.GetValueOrDefault(260 / 4));
```

Comment:
`260 / 4` How come this? I don't get it.

Reply:
there is no way to know how many bytes a virtual byte is in this context so, we could think that 4 virtual bytes are more or less equivalent to 1 real byte. I mean, the mempool size is in real bytes while everything else is expressed in virtual bytes, we need a way to "convert" virtual bytes to real bytes or real bytes to virtual byte.